### DISCUSSION TITLE
Evaluating common lisp with output sent to repl in sly-connected-hook

### DISCUSSION THREAD

**Discussion by ormf** [joaotavora/sly#660] [Q&A]
After loading a sbcl image I'd like to automatically evaluate init code in the sbcl repl after the slynk connection is established. Using sly-interactive-eval in a sly-connected-hook will evaluate the code, but the output is collected in the kill-ring rather than printed to the repl. In addition changing the package in the evaluated function will not be reflected in the repl either. Does anybody know how to get this done?

**Comment by aadcg** [comment]
You could add it to SBCL's init file at `~/.sbclrc`.
